scripts/test-manual-e2e.sh

```diff
@@ -24,98 +24,139 @@ info() {
+    ./gradlew :ReactAndroid:installArchives || error "Couldn't generate artifacts"
 
-npm pack
-
-PACKAGE=$(pwd)/react-native-$PACKAGE_VERSION.tgz
-success "Package bundled ($PACKAGE)"
-
-node scripts/set-rn-template-version.js "file:$PACKAGE"
-success "React Native version changed in the template"
-
-project_name="RNTestProject"
-
-cd /tmp/
-rm -rf "$project_name"
-node "$repo_root/cli.js" init "$project_name" --template "$repo_root"
-
-info "Double checking the versions in package.json are correct:"
-grep "\"react-native\": \".*react-native-$PACKAGE_VERSION.tgz\"" "/tmp/${project_name}/package.json" || error "Incorrect version number in /tmp/${project_name}/package.json"
-grep -E "com.facebook.react:react-native:\\+" "${project_name}/android/app/build.gradle" || error "Dependency in /tmp/${project_name}/android/app/build.gradle must be com.facebook.react:react-native:+"
-
-success "New sample project generated at /tmp/${project_name}"
-
-info "Test the following on Android:"
-info "   - Disable Fast Refresh. It might be enabled from last time (the setting is stored on the device)"
-info "   - Verify 'Reload JS' works"
-info ""
-info "Press any key to run the sample in Android emulator/device"
-info ""
-read -n 1
-cd "/tmp/${project_name}" && npx react-native run-android
+    kill_packagers
+}
 
-info "Test the following on iOS:"
-info "   - Disable Fast Refresh. It might be enabled from last time (the setting is stored on the device)"
-info "   - Verify 'Reload JS' works"
-info "   - Test Chrome debugger by adding breakpoints and reloading JS. We don't have tests for Chrome debugging."
-info "   - Disable Chrome debugging."
-info "   - Enable Fast Refresh, change a file (index.js) and save. The UI should refresh."
-info "   - Disable Fast Refresh."
-info ""
-info "Press any key to open the project in Xcode"
-info ""
-read -n 1
-open "/tmp/${project_name}/ios/${project_name}.xcodeproj"
+# Test RNTester
+{
+    info "Starting the packager in the background"
+    node "$CLI_PATH" start &
+
+    # Android
+    {
+        ./gradlew :RNTester:android:app:installJscDebug || error "Couldn't build RNTester Android"
+
+        info "Deleting previously installed Android RNTester app"
+        adb uninstall com.facebook.react.uiapp
+
+        info "Press any key to run RNTester in an already running Android emulator/device"
+        info ""
+        read -n 1
+        adb shell am start -n com.facebook.react.uiapp/.RNTesterActivity
+    }
+
+    # iOS
+    pushd RNTester
+    {
+        info "Press any key to open the workspace in Xcode, then build and test manually."
+        info ""
+        read -n 1
+        success "Installing CocoaPods dependencies"
+        rm -rf Pods && pod install
+        open "RNTesterPods.xcworkspace"
+    }
+    popd
+
+    info "When done testing RNTester app on iOS and Android press any key to continue."
+    info ""
+    read -n 1
+
+    kill_packagers
+}
 
-cd "$repo_root"
+# Test new app from template
+{
+    npm pack
```

Comment:
```suggestion
    yarn pack
```
(or is it a npm specific command?)

Reply:
Not really, but it also doesn’t do anything with version resolving, so I’m less worried about it. I can take a look, though, and make the whole script uniform 👍